code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -488,13 +486,13 @@
 	description = "A popular food paste made from ground dry-roasted peanuts."
 	color = "#C29261"
 	value = 3
-	nutriment_factor = 15 * REAGENTS_METABOLISM
+	nutriment_factor = 10 * REAGENTS_METABOLISM
```

Comment:
this is actual food

Reply:
Its a plant you juice giving you a lot of food for little work.
I know its a fatty and high-food count thing irl and such so its not eating to big of a nerf but still were trying to avoid such things